internal/command/command.go

```diff
@@ -129,6 +131,7 @@ func StartCommands(es *eventstore.Eventstore,
 
 func AppendAndReduce(object interface {
 	AppendEvents(...eventstore.Event)
+	// TODO: Why is it allowed to return an error here?
```

Comment:
@adlerhurst, reduce functions should always succeed, right?

Reply:
yes it's bad practice from the past. It's addressed in the refactoring of eventstore